### ISSUE TITLE
How to get cg.pdb file

### ISSUE THREAD

**Issue by hliu56** [ksy141/mstool#2]
Hi,

I wonder how do you get the cg.pdb files for those examples showed in your documentation?

Is it obtained from the protein databank or after running coarse grain simulation?

Thank you!

**Comment by ksy141** [comment]
Hi,There should be several examples (cg.pdb) in $root/src/mstool/examples/BackmappingBest,SiyoungOn Oct 16, 2024, at 6:39 PM, hliu56 ***@***.***> wrote:﻿
Hi,
I wonder how do you get the cg.pdb files for those examples showed in your documentation?
Is it obtained from the protein databank or after running coarse grain simulation?
Thank you!

**Comment by hliu56** [comment]
Yes, I've seen those cg.pdb files in examples.

But if I have some new lipids, how can I generate those files? Do those cg.pdb files obtained from literature or from coarse grain simulation?

Thank you!

**Comment by ksy141** [comment]
Do you want to use it as a membrane builder or backmapping tool? The later already assumes that you have a coarse-grained structure, so I guess you want to include a new lipid in the membrane builder?Best,SiyoungOn Oct 16, 2024, at 11:46 PM, hliu56 ***@***.***> wrote:﻿
Yes, I've seen those cg.pdb files in examples.
But if I have some new lipids, how can I generate those files? Do those cg.pdb files obtained from literature or from coarse grain simulation?
Thank you!

**Comment by hliu56** [comment]
Yes, I want to include some new lipids. For example, in Example 5. Martini bilayer, I may have a new set of lipids instead of lipids provided in the repository. So, I wonder how the coarse-grained structures of new lipids can be obtained/generated?

Thank you!

**Comment by ksy141** [comment]
Coarse-grained structures of new lipids in membrane building are not required. You need all-atom force field, coarse-grained force field, and mapping and isomeric information of the new lipid.Best,SiyoungOn Oct 17, 2024, at 12:12 AM, hliu56 ***@***.***> wrote:﻿
Yes, I want to include some new lipids. For example, in Example 5. Martini bilayer, I may have a new set of lipids instead of lipids provided in the repository. So, I wonder how the coarse-grained structures of new lipids can be obtained/generated?
Thank you!

**Comment by hliu56** [comment]
Thank you very much for the explanation!
